tests/data_context/store/test_datasource_store.py

```diff
@@ -335,20 +335,6 @@ def test_datasource_store_retrieve_by_name(
     )
 
 
-@pytest.mark.unit
-def test_datasource_store_delete_by_name(
```

Comment:
Should we add back in a unit test for the delete method? Or do we already have that somewhere else? (Besides the e2e test I mean)

Reply:
good call, added a unit test!